config/dompdf.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+return array(
+
+    /*
+    |--------------------------------------------------------------------------
+    | Settings
```

Comment:
tl;dr assuming these are defaults?

Reply:
It's the file from the ARCVStore system.

Reply:
:small_airplane: